AirCasting/Settings/MyAccountViewSignOut.swift

```diff
@@ -8,6 +8,7 @@ struct MyAccountViewSignOut: View {
     let logoutController: LogoutController
     @EnvironmentObject var userAuthenticationSession: UserAuthenticationSession
     @EnvironmentObject var persistenceController: PersistenceController
+    @Binding var isShowingLoadingScreen: Bool
```

Comment:
shouldn't we add initial value here instead of having to pass it in via constructor?